tests/unit/s2n_record_size_test.c

```diff
@@ -107,6 +107,9 @@ int main(int argc, char **argv)
 
     #define ONE_BLOCK 1024
     #define ONE_HUNDRED_K 100000
+    #define RECORD_SIZE_HIGH 3
+    #define RECORD_SIZE_LOW 4
```

Comment:
Is there a more descriptive name you could give here?

Reply:
hmm does HIGH_BYTE_ORDER/LOW_BYTE_ORDER sounds better?